src/everest/util/__init__.py

```diff
@@ -76,11 +77,38 @@ def makedirs_if_needed(path, roll_if_exists=False):
     os.makedirs(path)
 
 
+def warn_user_that_runpath_is_nonempty() -> None:
+    question = (
+        "You choose to run Everest in an existing non-empty runpath.\n"
+        "Please be aware of the following:\n"
+        "- Previously generated results "
+        "might be overwritten.\n"
+        "- Previously generated files might "
+        "be used if not configured correctly.\n"
+        "Are you sure you want to continue? Y/N : "
+    )
+    if not _get_user_confirmation(question):
+        sys.exit(
+            "Everest terminated execution due to existing runpath folder.\n"
+            "Please manually delete or change simulation_folder path in the config.yml"
+        )
+
+
+def _get_user_confirmation(message: str) -> bool:
```

Comment:
Nitpick: Would put this function inside `warn_user_that_runpath_is_nonempty` or just inline its logic into `warn_user_that_runpath_is_nonempty` entirely